### PR TITLE
Add check-packaging

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -59,7 +59,11 @@ Tracker = "https://github.com/BlueBrain/NeuroM/issues"
 neurom = 'neurom.apps.cli:cli'
 
 [tool.setuptools.packages.find]
-include = ["neurom"]
+where = ["."]
```
Comment (mgeplf) [inline pyproject.toml:62]: did you check the sdist to see everything worked?

Comment (adrien-berchet) [reply pyproject.toml:62]: I think it should be `where = ["neurom*"]` to avoid including tests and other random stuffs.

Comment (eleftherioszisis) [reply pyproject.toml]: Changed to that. Both include and where seem to work. With the latest changes I get:
wheel:
```
.
├── apps
│   ├── annotate.py
│   ├── cli.py
│   ├── config
│   │   ├── morph_check.yaml
│   │   └── morph_stats.yaml
│   ├── __init__.py
│   ├── morph_check.py
│   └── morph_stats.py
├── check
│   ├── __init__.py
│   ├── morphology_checks.py
│   ├── morphtree.py
│   ├── neuron_checks.py
│   └── runner.py
├── core
│   ├── dataformat.py
│   ├── __init__.py
│   ├── morphology.py
│   ├── neuron.py
│   ├── population.py
│   ├── soma.py
│   └── types.py
├── features
│   ├── bifurcationfunc.py
│   ├── bifurcation.py
│   ├── __init__.py
│   ├── morphology.py
│   ├── neurite.py
│   ├── population.py
│   ├── sectionfunc.py
│   └── section.py
├── geom
│   ├── __init__.py
│   └── transform.py
├── io
│   ├── __init__.py
│   └── utils.py
├── neurom-3.2.9.dev5.dist-info
│   ├── AUTHORS.md
│   ├── entry_points.txt
│   ├── LICENSE.txt
│   ├── METADATA
│   ├── RECORD
│   ├── top_level.txt
│   └── WHEEL
└── view
    ├── dendrogram.py
    ├── __init__.py
    ├── matplotlib_impl.py
    ├── matplotlib_utils.py
    └── plotly_impl.py
```
source tar:
```
.
├── asv.conf.json
├── AUTHORS.md
├── CHANGELOG.rst
├── doc
│   ├── clean.sh
│   ├── make.bat
│   ├── Makefile
│   └── source
│       ├── api.rst
│       ├── changelog.rst
│       ├── cli.rst
│       ├── conf.py
│       ├── definitions.rst
│       ├── developer.rst
│       ├── documentation.rst
│       ├── examples.rst
│       ├── features.rst
│       ├── images
│       │   ├── spherical_coordinates.svg
│       │   └── spherical_coordinates.tex
│       ├── index.rst
│       ├── install.rst
│       ├── license.rst
│       ├── logo
│       │   └── NeuroM.jpg
│       ├── migration.rst
│       ├── morph_check.rst
│       ├── morph_stats.rst
│       ├── quickstart.rst
│       ├── spherical_coordinates.rst
│       ├── tutorial.rst
│       └── validation.rst
├── examples
│   ├── boxplot.py
│   ├── density_plot.py
│   ├── end_to_end_distance.py
│   ├── extract_distribution.py
│   ├── features_graph_table.py
│   ├── get_features.py
│   ├── histogram.py
│   ├── iteration_analysis.py
│   ├── nl_fst_compat.py
│   ├── plot_somas.py
│   ├── radius_of_gyration.py
│   ├── section_ids.py
│   └── soma_radius_fit.py
├── LICENSE.txt
├── MANIFEST.in
├── neurom
│   ├── apps
│   │   ├── annotate.py
│   │   ├── cli.py
│   │   ├── config
│   │   │   ├── morph_check.yaml
│   │   │   └── morph_stats.yaml
│   │   ├── __init__.py
│   │   ├── morph_check.py
│   │   └── morph_stats.py
│   ├── check
│   │   ├── __init__.py
│   │   ├── morphology_checks.py
│   │   ├── morphtree.py
│   │   ├── neuron_checks.py
│   │   └── runner.py
│   ├── core
│   │   ├── dataformat.py
│   │   ├── __init__.py
│   │   ├── morphology.py
│   │   ├── neuron.py
│   │   ├── population.py
│   │   ├── soma.py
│   │   └── types.py
│   ├── exceptions.py
│   ├── features
│   │   ├── bifurcationfunc.py
│   │   ├── bifurcation.py
│   │   ├── __init__.py
│   │   ├── morphology.py
│   │   ├── neurite.py
│   │   ├── population.py
│   │   ├── sectionfunc.py
│   │   └── section.py
│   ├── geom
│   │   ├── __init__.py
│   │   └── transform.py
│   ├── __init__.py
│   ├── io
│   │   ├── __init__.py
│   │   └── utils.py
│   ├── morphmath.py
│   ├── neurom.egg-info
│   │   ├── dependency_links.txt
│   │   ├── entry_points.txt
│   │   ├── PKG-INFO
│   │   ├── requires.txt
│   │   ├── SOURCES.txt
│   │   └── top_level.txt
│   ├── stats.py
│   ├── utils.py
│   ├── view
│   │   ├── dendrogram.py
│   │   ├── __init__.py
│   │   ├── matplotlib_impl.py
│   │   ├── matplotlib_utils.py
│   │   └── plotly_impl.py
│   └── viewer.py
├── PKG-INFO
├── pylintrc
├── pyproject.toml
├── README.md
├── setup.cfg
├── setup.py
└── tox.ini
```

Comment (eleftherioszisis) [reply pyproject.toml]: Only `include=["neurom"]` seems to be doing the trick..

**Review by mgeplf — APPROVED** [review]

**Pull request merged**